cudaaligner/benchmarks/main.cpp

```diff
@@ -78,9 +80,11 @@ class CudaStream
 template <typename AlignerT>
 static void BM_SingleBatchAlignment(benchmark::State& state)
 {
+    const std::size_t max_gpu_memory = cudautils::find_largest_contiguous_device_memory_section();
     CudaStream stream;
-    int32_t alignments_per_batch = state.range(0);
-    int32_t genome_size          = state.range(1);
+    DefaultDeviceAllocator allocator = create_default_device_allocator(max_gpu_memory);
```

Comment:
Why was the default value ok above and `max_gpu_memory` is needed here?

Reply:
The first benchmark does just a single alignment which does not require much memory. This benchmark does batched alignments and requires as much memory as it can get.

Reply:
Any follow up question or comment on this?

Reply:
No, no follow up questions.